src/App.tsx

```diff
@@ -1,24 +1,209 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { Todo } from './types/Todo';
+import {
+  deleteTodo, getTodos, postTodos, updateTodo,
+} from './api/todos';
+import { CurrentError } from './types/CurrentError';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { ErrorNotification } from
+  './components/ErrorNotification/ErrorNotification';
+import { TodoFilter } from './types/TodoFilter';
+import { filterTodos } from './utils/filter';
+import { ToggleType } from './types/ToggleType';
 
-const USER_ID = 0;
+const USER_ID = 11535;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
```

Comment:
Do you really need a state for filteredTodos? You may just use constant (this variable depends on todoFilter)